app/views/sufia/admin/admin_sets/_form.html.erb

```diff
@@ -24,6 +24,10 @@
                   <%= f.input :thumbnail_id, collection: @form.select_files %>
                 <% end %>
 
+                <% if @form.persisted? %>
```

Comment:
Ok - Will work on that. The problem I was getting before was the the permission template wasn't created, so the workflow_name was getting lost somewhere. I'll see what I can do.